hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/FileIOEngine.java

```diff
@@ -100,7 +96,6 @@ public FileIOEngine(long capacity, boolean maintainPersistence, String... filePa
         LOG.info("Allocating cache " + StringUtils.byteDesc(sizePerFile)
             + ", on the path:" + filePath);
       } catch (IOException fex) {
-        LOG.error("Failed allocating cache on " + filePath, fex);
```

Comment:
below thrown exception isn't going to necessarily include the information that we failed to allocate the cache on the specific path?